libraries/AP_GyroFFT/AP_GyroFFT.h

```diff
@@ -222,11 +226,15 @@ class AP_GyroFFT
         // fft engine output rate
         uint32_t _output_cycle_ms;
         // tracked frequency peak
-        Vector3<uint8_t> _tracked_peak;
+        Vector3<uint32_t> _tracked_peak;
```

Comment:
should this be a Vector3 of enum FrequencyPeak?

Reply:
maybe enum FrequencyPeak needs a :uint8_t to make it uint8_t base type

Reply:
I could do that, but more flash because more template methods defined?

Reply:
I've made the base type change but setting it to Vector3<FrequencyPeak> causes other typing issues, so I left it as uint8_t